src/services/createRouterRoute.ts

```diff
@@ -1,18 +1,23 @@
+import { reactive, toRefs } from 'vue'
 import { ResolvedRoute } from '@/types/resolved'
+import { ResolvedRouteQuery } from '@/types/resolvedQuery'
 import { RouterPush, RouterPushOptions } from '@/types/routerPush'
 import { RouteUpdate } from '@/types/routeUpdate'
 import { Writable } from '@/types/utilities'
 
 const isRouterRouteSymbol = Symbol('isRouterRouteSymbol')
 
-export type RouterRoute<TRoute extends ResolvedRoute = ResolvedRoute> = Omit<ResolvedRoute, 'params'> & Readonly<{
+export type RouterRoute<TRoute extends ResolvedRoute = ResolvedRoute> = Readonly<{
+  key: TRoute['key'],
+  matched: TRoute['matched'],
+  matches: TRoute['matches'],
+  query: ResolvedRouteQuery,
   params: Writable<TRoute['params']>,
   update: RouteUpdate<TRoute>,
-  [isRouterRouteSymbol]: true,
```

Comment:
This was the problem that kept popping up in the build. We don't actually want this symbol in the types. We just want to use it internally. Removing it from this type fixes the issue. Which makes total sense because this is a public type and `isRouterRouteSymbol` was not exported.

Reply:
nice job on this, that was certainly stressful to see intermittently.